api/compute/v1alpha1/function_webhook.go

```diff
@@ -194,6 +194,23 @@ func (r *Function) ValidateCreate() error {
 		allErrs = append(allErrs, fieldErrs...)
 	}
 
+	if r.Spec.Pod.VPA != nil {
+		if r.Spec.MaxReplicas != nil {
+			allErrs = append(allErrs,
+				field.Invalid(field.NewPath("spec").Child("pod").Child("vpa"), r.Spec.Pod.VPA, "you can not enable HPA and VPA at the same time"))
```

Comment:
> Vertical Pod Autoscaler should not be used with the [Horizontal Pod Autoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-resource-metrics) (HPA) on CPU or memory at this moment. However, you can use VPA with [HPA on custom and external metrics](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-custom-metrics).

Do we plan to support this kind of mixed usage in the future?

Reply:
there is no such plan yet, but I think there would be one